services/skus/controllers.go

```diff
@@ -92,7 +98,9 @@ func Router(
 	)
 
 	// Receipt validation.
-	r.Method(http.MethodPost, "/{orderID}/submit-receipt", metricsMwr("SubmitReceipt", corsMwrPost(SubmitReceipt(svc))))
+	valid := validator.New()
```

Comment:
I agree, but unfortunately this is what the package is called. In my experience with using this package (which has been a long time), it has not clashed with other things.

If it's confined to where the validation should take place, i.e. at the transport layer, before entering the business logic territory, it might be OK. Like the other example, in the `handler` package, it's clear what it is for.

With this PR, I considered placing the new code in `handler`, but that would also involve moving `SubmitReceipt` there. In itself this is not a problem. The problem is that it would also involve moving `HandleWebhook` for both, iOS and Android, as those functions share code with `SubmitReceipt`.

It would have been fine, had it not meant that `handler` would have to import `skus` – which is best avoided, as `skus` already (temporarily) imports `handler`. Or, it would mean moving around more code to `model` and more refactoring to `Servcie`, which, at this stage of the project, seemed like not the price to be paid right now (despite the fact that I wanted and will do it at some point).

So for now, I've moved it to an explicit scope to, at least, avoid clashing with possible other names. Another option would be to rename the package to `datavalidator` or  something, but this does not sound appealing to me – it would require us and other committers remember to name it the same way everywhere, which could easily get out of hand. Not to mention that it gets unreasonably long.

Let me know if the updated version looks better.